libraries/freertos_plus/aws/ota/src/aws_iot_ota_agent.c

```diff
@@ -2410,7 +2487,7 @@ static IngestResult_t prvIngestDataBlock( OTA_FileContext_t * C,
     }
 
     /* Close the file and cleanup.*/
-    if( eIngestResult == eIngest_Result_Uninitialized )
+    if( eIngestResult == eIngest_Result_Accepted_Continue )
```

Comment:
Is `eIngest_Result_Accepted_Continue == eIngest_Result_Uninitialized`?  Seems like we've changed the logic here based on the new name.

Reply:
this is an error made from Prasad previously, I'm correcting it here, see this commit, b5f218a5fbd905ff9624c44f958b49715064092b